main.go

```diff
@@ -283,8 +280,16 @@ func main() {
 		os.Exit(1)
 	}
 
+	setupLog.Info("setting up webhook server TLS config")
+	webhookServer := mgr.GetWebhookServer()
+	if webhookServer == nil {
+		setupLog.Error(err, "unable to get controller manager webhook server")
```

Comment:
May be clearer to pass an explicit `nil` here instead. That said, I think we can safely depend on `mgr. GetWebhookServer()` not ever returning `nil` 'cause the manager implementation will panic, and if it would return nil effect is the ultimately the same

Reply:
Makes sense. Updated to remove the nil check.